code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -18,7 +20,43 @@
 /obj/item/clothing/mask/muzzle/attack_hand(mob/user as mob)
 	if(user.wear_mask == src && !user.IsAdvancedToolUser())
 		return 0
+	else if(user.wear_mask == src && security_lock && locked)
+		if(ishuman(user))
+			if(doUnlock(user))
+				visible_message("<span class='danger'>[user] unlocks their [src.name].</span>", \
+									"<span class='userdanger'>[user] unlocks their [src.name].</span>")
+		else
+			to_chat(user, "You lack the ability to open the lock.")

```

Comment:
All of these `to_chat(`s are missing span classes, completely un-formatted text looks bad. I'd advise using `<span class='warning'>`